src/equations/compressible_euler_multicomponent_2d.jl

```diff
@@ -252,44 +600,44 @@ function initial_condition_weak_blast_wave(x, t, equations::CompressibleEulerMul
   phi               = atan(y_norm, x_norm)
   sin_phi, cos_phi  = sincos(phi)
 
-  # Calculate primitive variables.    !!! NOTE: EC flux does not transfer mass across an interface separating two different species
-  rho1              = r > 0.5 ? 0.2*1.0 : 0.2*1.1691
-  rho2              = r > 0.5 ? 0.8*1.0 : 0.8*1.1691
-  rho               = rho1 + rho2
+  prim_rho          = SVector{NCOMP, real(equations)}(r > 0.5 ? 2^(i-1) * (1-2)/(1-2^NCOMP)*1.0 : 2^(i-1) * (1-2)/(1-2^NCOMP)*1.1691 for i = 1:NCOMP)
+
   v1                = r > 0.5 ? 0.0 : 0.1882 * cos_phi
   v2                = r > 0.5 ? 0.0 : 0.1882 * sin_phi
   p                 = r > 0.5 ? 1.0 : 1.245
 
-  return prim2cons(SVector(rho1, rho2, v1, v2, p), equations)
+  prim_else         = SVector{3, real(equations)}(v1, v2, p)
```

Comment:
Sorry for only seeing this now. I don't think the variable name `prim_else` is optimal, since `else` implies `otherwise`. Also, at least to me `else` has a *very* particular meaning in the context of programming, and it feels weird to see this keyword as part of a variable name. Thus I'd prefer something like `prim_other` (or something else); the same goes for other occurrences of "else" as part of a variable name. However, this is not a breaking issue, thus if you feel strongly about keeping `else` in the name, I can learn to live with it ;-)

Reply:
done in c7e9a75